libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/model/eh/Edge.kt

```diff
@@ -52,7 +53,8 @@ class Edge internal constructor(
     val speedLimit: Double?,
     val laneCount: Byte?,
     val meanElevation: Double?,
-    val countryCode: String?,
+    val countryCodeIso2: String?,
+    val countryCodeIso3: String?,
```

Comment:
For consistency, could we change the order here? In 2.0 we have `countryCodeIso3` first 👀 

https://github.com/mapbox/mapbox-navigation-android/blob/f4e3e9c2fd7cf4cda7401609807af3ff0dc548b2/libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/model/eh/EHorizonEdgeMetadata.kt#L43-L44